src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/21.01/analysis-submission-email-pipeline-result-on-error.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="analysis-submission-email-pipeline-result-on-error" author="deep">
+        <addColumn tableName="analysis_submission">
+            <column name="email_pipeline_result_error" type="bit(1)" valueBoolean="false"
+                    defaultValueBoolean="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+
+        <addColumn tableName="analysis_submission_AUD">
+            <column name="email_pipeline_result_error" type="bit(1)" />
+        </addColumn>
+
+        <!--Updating analysis submission table to change column email_pipeline_result to email_pipeline_result_completed -->
+        <sql>
+            ALTER TABLE analysis_submission CHANGE email_pipeline_result email_pipeline_result_completed BIT(1) not null DEFAULT 0;
```

Comment:
Can you use the renameColumn thing fom liquibase instead of the custom sql?  https://docs.liquibase.com/change-types/community/rename-column.html

Reply:
Sure thing!

Reply:
Updated in 741ea9aed7fbdd9feb951ac92bc700b0eea52e04